docs/source/tutorial/first_steps.rst

```diff
@@ -22,6 +22,54 @@ that you can use :func:`torch.load` to load a model like:
 More information on PyTorch's model persistence can be found at:
 https://pytorch.org/tutorials/beginner/saving_loading_models.html.
 
+Using Learned Embeddings
+------------------------
+The embeddings learned for entities and relations are useful for link
+prediction in PyKEEN, but also generally useful for other downstream
+machine learning tasks like clustering, regression, and classification.
+
+The embeddings themselves are typically stored in a :class:`pykeen.nn.Embedding`,
+which wraps the :class:`torch.nn.Embedding` with several key values. They
+can be accessed like this:
+
+.. code-block:: python
+
+    from pykeen.pipeline import pipeline
+    result = pipeline(model='TransE', dataset='UMLS')
+    model = result.model
+    entity_embeddings = model.entity_embeddings._embeddings.weight.data
```

Comment:
I would propose to directly use the "new-style" way, since it already works for all models.

Reply:
Yes that sounds good.